debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/naming/CustomColumnNamingStrategy.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.jdbc.naming;
+
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.connector.jdbc.util.NamingStyle;
+import io.debezium.connector.jdbc.util.NamingStyleUtils;
+
+/**
+ * A custom implementation of the {@link io.debezium.sink.naming.ColumnNamingStrategy} interface
+ * that supports various naming styles and customizations such as snake_case,
+ * camelCase, and prefix/suffix additions.
+ *
+ * @author Gustavo Lira
+ */
+public class CustomColumnNamingStrategy implements ColumnNamingStrategy {
```

Comment:
Same, I believe this is no longer needed as you've moved the code to a transform.